### DISCUSSION TITLE
프로젝트 워크플로우 명세

### DISCUSSION THREAD

**Discussion by seonyae-park** [seonyae-park/hello-charlie#6] [Ideas]
@ssheko @tjdrud1090 
찰리 프로젝트에서 제공할 UX 명세를 적어주세요🧤

**Comment by seonyae-park** [comment]
사용자별 캘린더 기능 
 - 다녀온 방탈출 리스트를 일자별로 표시
 - 일자 내의 아이템 선택시 테마 카드로 이동
 - 예약/방문을 예정할 수 있는 것도 추가


* 여러 사용자별 캘린더 공유 기능도....?!
* 테마명, 매장명 등 검색을 통한 바로가기

**Comment by seonyae-park** [comment]
사용자별 다녀온 테마 통계 보기 기능
* 통계: 성공율, 노힌트율, 달성율(지역별/전국)
* 정렬 - 일자순, 리뷰기록순, 평점순 등

**Comment by seonyae-park** [comment]
테마 검색 기능
- 이름 검색(포함)
- 검색 옵션: 장르, 지역 등

**Comment by seonyae-park** [comment]
사용자 로그인
- OAuth (카카오톡, 구글, 네이버?)

**Comment by seonyae-park** [comment]
리스트 보기 옵션
- 매장별 or 테마별

1. 테마별 
   - (코로리 기반의) 랭킹별
   - 이름순
   - (서버생성순)등록순

**Comment by seonyae-park** [comment]
테마 카드 (상세보기)
- 이름, 테마소개, 썸네일
- 매장명 (선택시 매장 카드 팝업), 지역
- 시간, 난이도, 활동성, 공포도, 추천여부
- 리뷰 목록 
  - 리뷰 작성 버튼

* 내가 작성한 리뷰에,
  * 리뷰 수정버튼
  * private 메모 추가/수정 기능 


리뷰 편하게 작성하도록 하기
- 테마 카드를 long-press 하면 간단한 리뷰 작성 다이얼로그 팝업? 
- (별점으로 각 항목을 빠르게 평가하는 정도로만 제공)

**Comment by seonyae-park** [comment]
매장 카드
- 이름
- 테마 목록
- 지역, 위치(지도)

**Comment by seonyae-park** [comment]
마이 페이지
- 프로필
- 내 방탈출 목록
  - [내 통계 보기](https://github.com/seonyae-park/hello-charlie/discussions/6#discussioncomment-10096011)
  - 정렬 - 테마순, 매장순, 평점순, 날짜순, 힌트순 등
  - ...

**Comment by tjdrud1090** [comment]
Notion 문서로 따로 정리하였습니다. 확인 부탁드려요~:)
